README.md

```diff
@@ -45,6 +45,62 @@ Table with 4 columns and 2 rows:
 
 Please see the to-do list above for project ideas as well as any open issues! If you add new functionality, please add appropriate documentation and testing. In addition, please increment the minor version of the package to reflect the new changes!
 
+Tests are run with [ParallelTestRunner.jl](https://github.com/JuliaTesting/ParallelTestRunner.jl). See below for some brief usage examples:
+
+**`Pkg.test()` workflow in root directory:**
+
+```julia-repl
+julia --proj --threads=auto
+
+(Photometry) pkg> test
+```
+**Interactive workflow in `test` directory:**
+
+```julia-repl
+julia --proj --threads=auto
+
+julia> using Photometry, ParallelTestRunner
+```
+
+```julia-repl
+# List available testsets
+
+julia> runtests(Photometry, ["--list"])
+Available tests:
+ - aperture/circular
+ - aperture/elliptical
+ - aperture/overlap
+ - aperture/photometry
+ - aperture/plotting
+ - aperture/rectangle
+ - backgroud/background
+ - backgroud/estimators
+ - backgroud/interpolators
+ - detection/detection
+```
+
+```julia-repl
+# Run a subset of tests
+
+julia> const init_code = quote
+           import StatsBase: median, mean, std, mad
+
+           const DATA_DIR = joinpath(@__DIR__, "data")
+       end
+
+julia> runtests(Photometry, ["--verbose"]; init_code, test_filter = test -> occursin("aperture", test))
```

Comment:
Why not using Pkg.test? You don't need to do all the above.

Reply:
> If you think ParallelTestRunner documentation could be improved to make that clearer

Maybe it needs a section for those using the tests, in addition to the one about setting them up?

Reply:
Thanks! Just opened https://github.com/JuliaTesting/ParallelTestRunner.jl/pull/60 to discuss